packages/fast-components-react-msft/src/action-toggle/action-toggle.spec.tsx

```diff
@@ -0,0 +1,291 @@
+            unselectedLabel: "unselected",
+        };
+        const onChange: any = jest.fn();
+        const rendered: any = mount(
+            <ActionToggle
+                className={"custom-class-name"}
+                {...props}
+                onChange={onChange}
+            />
+        );
+
+        rendered.find("button").simulate("click");
+
+        expect(onChange).toHaveBeenCalledTimes(1);
+
+        rendered.find("button").simulate("click");
+
+        expect(onChange).toHaveBeenCalledTimes(2);
+    });
+
+    test("should apply selected class after click event", () => {
+        const props: ActionToggleHandledProps = {
+            selectedLabel: "selected",
+            unselectedLabel: "unselected",
+        };
+        const onChange: any = jest.fn();
+        const rendered: any = mount(
+            <ActionToggle
+                className={"custom-class-name"}
+                {...props}
+                onChange={onChange}
+            />
+        );
+
+        expect(rendered.find("button").prop("className")).not.toContain(
+            "actionToggle__selected"
+        );
+        rendered.find("button").simulate("click");
+        expect(rendered.find("button").prop("className")).toContain(
+            "actionToggle__selected"
+        );
+    });
+
+    test("should not apply selected class after click event when props specify selected = false", () => {
+        const props: ActionToggleHandledProps = {
+            selectedLabel: "selected",
+            unselectedLabel: "unselected",
+            selected: false,
+        };
+        const onChange: any = jest.fn();
+        const rendered: any = mount(
+            <ActionToggle
+                className={"custom-class-name"}
+                {...props}
+                onChange={onChange}
+            />
+        );
+
+        expect(rendered.find("button").prop("className")).not.toContain(
+            "actionToggle__selected"
+        );
+        rendered.find("button").simulate("click");
+        expect(rendered.find("button").prop("className")).not.toContain(
+            "actionToggle__selected"
+        );
+    });
+
+    test("should apply a hasGlyphAndContent class when display is content only", () => {
+        const props: ActionToggleHandledProps = {
+            selectedLabel: "selected",
+            unselectedLabel: "unselected",
+            selectedContent: "selected",
+            unselectedContent: "unselected",
+        };
+
+        const rendered: any = mount(<ActionToggle {...props} />);
+        expect(rendered.find("button").prop("className")).not.toContain(
+            "actionToggle__hasGlyphAndContent"
+        );
+
+        rendered.find("button").simulate("click");
+        expect(rendered.find("button").prop("className")).not.toContain(
+            "actionToggle__hasGlyphAndContent"
+        );
+    });
+
+    test("should not apply a hasGlyphandContent  class when display is glyph only", () => {
```

Comment:
Some extra spacing in the description

Reply:
Not seeing resolution or reasoning for not resolving

Reply:
Updated with changes.